bats/fb-content-katello.bats

```diff
@@ -175,6 +175,7 @@ setup() {
 }
 
 @test "check available errata" {
+  skip
```

Comment:
Generally I like to have a comment why it's skipped. Ideally a BZ so we can see when it's fixed.

Reply:
Added